web/packages/core/src/internal/i18n.ts

```diff
@@ -20,6 +21,7 @@ const bundles: Record<string, FluentBundle> = {};
 for (const [locale, files] of Object.entries(BUNDLED_TEXTS)) {
     const bundle = new FluentBundle(locale);
     if (files) {
+        const customMap = resetCustomMap();
```

Comment:
Do we think the possibility of the custom map getting defined as the loop is running makes it worth running this code inside the loop instead? The `restoreCustomMap` can still run outside the loop, since it wouldn't make sense to repeatedly restore it just to re-create a fresh Map on each iteration.

Reply:
Or do we think that possibility is remote enough to not be a concern?